nautobot/ipam/views.py

```diff
@@ -939,18 +939,14 @@ def get(self, request, *args, **kwargs):
     def post(self, request):
         interface, _ = retrieve_interface_or_vminterface_from_request(request)
 
-        if pks := request.POST.getlist("pk"):
+        pks = request.POST.getlist("pk")
+        if pks:
```

Comment:
Did this change need to be made? I thought the `:=` operator basically did the same thing as a one-liner as what you're doing now.